docs/source/release_roadmap.org

```diff
@@ -0,0 +1,363 @@
+#+TITLE: Recommendations regarding the release roadmap of Iroha v2
+#+AUTHOR: Aleksandr Petrosyan
+#+DATE: 2023-08-13
+
+* State of the project
+
+Iroha v2 is a capable ledger with notable subroutines that allow it to
+be deployed successfully as a private blockchain ledger:
+1. It has relatively good performance characteristics regarding
+   throughput (post RC9).
+2. It has good performance characteristics regarding transaction
+   finality and latency (post RC19 also good query latency).
+3. It has support for Turing complete permission validation, which
+   allows Iroha to enforce arbitrarily complex invariants on-chain. 
+4. It has support for a small interpreted VM called the Iroha Special
+   Instructions which allow small-footprint transactions. 
+5. It has support for Turing complete smart-contracts, with an
+   underlying event driven reactive system, which was successfully
+   deployed to operate a cross-border delivery versus payment
+   implementation for ADB. 
+6. The ledger has three SDKs within the JVM, JavaScript and Python
+   ecosystems, allowing the users to easily adapt the Iroha node to
+   their requirements.
+7. The ledger is accompanied by a suite of tools: =kagami=,
+   =iroha_client_cli=, /etc/., which comprise a small SDK. 
+
+The ledger has been demonstrated to work during the ADB CSD/RTGS
+linkages project, and is slated to be used in two projects internally
+by SORAMITSU which only utilise the private blockchain ledger
+functionality.
+
+The ledger can support un-sharded monolithic blockchains, with limited
+export capability (unlike Iroha v1, v2 uses an internal representation
+and doesn't use a database). Its consensus was designed with Byzantine
+fault tolerance in mind, however, the algorithm -- Sumeragi has some
+intrinsic flaws outlined in [[https://arxiv.org/pdf/1707.01873.pdf][an IBM Research article]], which is notable
```

Comment:
the paper doesn't mention any intrinsic flaws in B-Chain anywhere in the paper

Reply:
not of BChain. I've added quotes to make it clearer what I mean.